js/pin.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+(function () {
+  var PinSize = {
+    HEIGHT: 70,
+    RADIUS: 50 / 2
+  };
+
+  var pinTemplate = document
+    .querySelector('#pin')
+    .content.querySelector('.map__pin');
+  window.renderPin = function (advert) {
```

Comment:
У тебя шаблон метки грубо наезжает на рендер, а ты не видишь!

```suggestion
  var pinTemplate = document.querySelector('#pin').content.querySelector('.map__pin');
  
  window.renderPin = function (advert) {
```